packages/protobuf-test/extra/name-clash.proto

```diff
@@ -15,6 +15,13 @@
 syntax = "proto3";
 package spec;
 
+import "extra/example.proto";
+
+// This message class will clash with the imported message class
+message User {
+  docs.User u = 1; // Reference the import with the same name to trigger a clash
+}
```

Comment:
We trigger the issue by declaring a message, and importing a message with the same name from a different namespace.